lambda/functions/log-data/lambda_function.py

```diff
@@ -0,0 +1,35 @@
+import boto3
+import ast
+import json
+import os
+from urllib.parse import unquote
+print('Loading function')
+
+def lambda_handler(event, context):
```

Comment:
Was terraform fmt applied to this file?

Reply:
no, as this is a python file

Reply:
Doh. It was too early in the morning when I looked at this.